index.html

```diff
@@ -12,15 +11,22 @@
     <title>My Portfolio</title>
 </head>
 <body>
-        <header class="main-header"> 
-            <a class="logo" href="#"><img src="images/My Logo (1).png" alt="home page logo"></a>
-            <nav id="nav-container">
-                    <a class="desk" href="#" >Project</a>
-                    <a class="desk" href="#" >About</a> 
-                    <a class="desk" href="#" >Contact</a>
-            <a class="menu" href="#" ><img src="icon/menu.png"alt="menu"></a>    
-        </nav>
-    </header>
+        <header>
+           <a class="logo" href="#"><img src="images/My Logo (1).png" alt="home page logo"></a>
+            <div class="menu-toggle">
+              <span class="bar"></span>
```

Comment:
- [ ] Kindly 😃as stated by the previous reviewer it will be better 💯 if you make the design like the Figma Design as I see that at the mobile menu, the header of the page not showing in the mobile menu at the Figma, but in your case, I see that the header it is existing in the mobile menu, so let's try to stick to The Figma to make it  nice like it 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/232999340-d70bb3c6-90ac-4324-8c17-b9ea759daa34.png) |  ![image](https://user-images.githubusercontent.com/83360636/232999514-4a5ce39f-b050-489d-b2e8-414049230329.png) |